staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -4492,7 +4509,17 @@ func (c *Cloud) findInstanceByNodeName(nodeName types.NodeName) (*ec2.Instance,
 // Returns the instance with the specified node name
 // Like findInstanceByNodeName, but returns error if node not found
 func (c *Cloud) getInstanceByNodeName(nodeName types.NodeName) (*ec2.Instance, error) {
-	instance, err := c.findInstanceByNodeName(nodeName)
+	var instance *ec2.Instance
+
+	// According to https://github.com/kubernetes/kubernetes/pull/78140 it worth the trade off to offload
```

Comment:
It is odd to see a self-referential comment in a PR. It would be better to have the explanation inline.

Reply:
done